Example/Controllers/ViewController.m

```diff
@@ -77,6 +77,12 @@ - (void)viewDidLoad {
         [weakSelf showAlertViewWithTitle:@"Globe"
                                  message:@"You have chosen the globe view controller."];
     } atIndex:4];
+    
+//    tabBarController.badgeFont = [UIFont systemFontOfSize:10 weight:2];
```

Comment:
Please let's don't leave commented lines of code here :)

We could instead leave a comment that says:
```
// For more customisation of badge labels, see properties in ESTabBarController.h.
```